tests/test_bitwise.py

```diff
@@ -67,50 +67,41 @@ def test_bitwise_and(self, lhs, rhs, dtype):
         )
         assert_array_equal(dp_a & dp_b, np_a & np_b)
 
-        """
```

Comment:
Do we have all other in-place functions which now are supported to be covered by the tests? Do we need to add more tests probably to secure in-place support? (like do we have testing in-place for `dpnp.remainder` and so on)

Reply:
Tests for in-place `remainder` and `floor_divide` is added to `tests/test_mathematical.py`